### ISSUE TITLE
please add `yalc pull` as alias for update

### ISSUE THREAD

**Issue by a7madgamal** [whitecolor/yalc#125]
you know, since we have `yalc push` already :)

**Comment by whitecolor** [comment]
any other relevant reason?

**Comment by a7madgamal** [comment]
nope, I hope thats relevant enough but in 2 different occasions I tried `yalc push` just because I used pull before, also adding it as an alias would be a harmless change IMO
